### ISSUE TITLE
[test-case-property-ordering] moved some properties to the last by default

### ISSUE THREAD

**Issue by mysticatea** [not-an-aardvark/eslint-plugin-eslint-plugin#79]
**Tell us about your environment**

- ESLint: master
- eslint-plugin-eslint-plugin: 2.0.1

**Please show your full configuration:**

- [.eslintrc.js](https://github.com/eslint/eslint/blob/2c7431d6b32063f74e3837ee727f26af215eada7/.eslintrc.js)

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
//...
        {
            code: "function f() { let a; let a; }",
            parser: looseParserPath,
            parserOptions: { ecmaVersion: 2015 },
            errors: [
                { message: "'a' is already defined.", type: "Identifier" }
            ]
        },
//....
```

```bash

```

**What did you expect to happen?**

No errors.

**What actually happened? Please include the actual, raw output from ESLint.**

![image](https://user-images.githubusercontent.com/1937871/54252523-8923ad00-458e-11e9-9825-204a893f861b.png)

----

According to [test-case-property-ordering.md](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/test-case-property-ordering.md), the default option doesn't seem to include some valid properties.

- `env`
- `filename`
- `globals`
- `parser`
- (`rules`)
- `settings`

**Comment by not-an-aardvark** [comment]
Thanks for creating an issue. I agree that this isn't ideal.

Currently, the rule requires that unspecified properties appear at the end of the object. Would it be better to make unspecified properties appear before `errors`?

If the user provides a custom options object, where `errors` is at the beginning, I'm not sure how best to handle unspecified properties in that case.

**Comment by mysticatea** [comment]
It's fine to move unknown properties to the last. If people are using a custom rule tester, they have to configure that ordering rule for additional properties. This is reasonable.

It's great if the rule recognizes the properties the official rule tester supports, by default :)

**Comment by not-an-aardvark** [comment]
I agree that we should change the default. However, this would be a breaking change.
